app/renderer/components/preferences/payment/history.js

```diff
@@ -113,82 +113,103 @@ class HistoryFooter extends ImmutableComponent {
       reconcileDate: formattedTimeFromNow(timestamp)
     }
 
-    return <div className={css(styles.historyFooter)}>
-      <div className={css(styles.nextPayment)}>
+    return <div className={css(styles.flexAlignCenter, styles.historyFooter)}>
+      <div className={css(styles.historyFooter__nextPayment)}>
         <span data-l10n-id={l10nDataId} data-l10n-args={JSON.stringify(l10nDataArgs)} />
       </div>
+      {/* TODO: refactor button.less */}
       <Button l10nId='paymentHistoryOKText'
-        className={css(commonStyles.primaryButton)}
+        className='primaryButton'
```

Comment:
As I wrote in the commit message:

> There have been already browserButton and primaryButton available with commonStyles.js, still these styles are overwritten by ones which button.less defines. We have to work on refactoring button.less at first.

It is because `aphrodite` was changed to `aphrodite/no-important`. I left the reasoning in the commit message on this change too.